docs/src/_tutorials/example-projects.md

```diff
@@ -3,9 +3,34 @@ title: Example Meltano Projects
 description: Learn how others are using Meltano by viewing their production Meltano projects.
 layout: doc
 ---
+If you have a production project that you'd like to add please open a [Pull Request](https://github.com/meltano/meltano/edit/main/docs/src/_tutorials/example-projects.md).
 
-The following Meltano projects are real-world examples of Meltano in production use. If you're stuck or just want to see how things fit together these projects are a great place to start.
+We group example projects into three categories:
+1. Running in production
+2. Simpler examples usually solving one problem
+3. Complete sandbox projects to play around with
 
-If you have a production project that you'd like to add please open a [Pull Request](https://github.com/meltano/meltano/edit/main/docs/src/_tutorials/example-projects.md).
+### 1. Running in production
+
+The following Meltano projects are real-world examples of Meltano in production use. If you're about to launch your pipelines
+into your own production environment or want to take a look at how others organize their projoects, these projects are a great place to start.
 
 - **[Meltano Squared](https://github.com/meltano/squared)** - This is the project the Meltano team uses to manage their Meltano instance.
```

Comment:
Thoughts on adding a few more notes here? We use essentially every feature of Meltano including environments, multiple yaml files, plugin inheritance, etc along with plugins like Great Expectations, SQLFluff, dbt, Airflow, Superset, all hosted on kubernetes.

Reply:
True, I added more notes, and another sandbox project while I'm at it.